src/HttpServer.ts

```diff
@@ -150,7 +249,29 @@ export class HttpServer {
         }
     }
 
-    async verifierHandler(req: Request, res: Response): Promise<void> {
+    /**
+     * @openapi
+     * /verifiers:
+     *   get:
+     *     summary: It returns the list of the trusted verifiers
+     *     description: "It returns the list of the trusted verifiers. 'Trusted' verifiers means that we trust `verifyRelayedCall` to be consistent: off-chain call and on-chain calls should either both succeed or both revert."
```

Comment:
i'm not sure about explicitly mentioning `verifyRelayedCall` if it is an internal function

Reply:
I thought that since it is provided by the `IRelayVerifier` interface, it's something the user should be aware of. If you think it may be confusing, we can rephrase this sentence.

Reply:
i think it's fine as it is :+1: